### PR TITLE
Docs: replacing "setting" to "option" in world docs 

### PR DESCRIPTION
**Pull request by ReverM**
## What is this fixing or adding?
Replacing every instances of "setting" to "option" in world docs (setup guide and info) when refering to player options
Replacing a stray instance of "setting" in option.py that was left in progression balancing description

## How was this tested?
Reading

To keep track of which game have been reviewed heres a list :p

- [x] Adventure
- [ ] LTTP
- [x] Archipidle
- [x] Sudoku
- [x] Blasphemous
- [x] Bumper Stickers
- [x] Checksfinder
- [x] Clique
- [x] DS3
- [x] DKC3
- [x] DLC Quest
- [x] DOOM
- [x] DOOM II
- [ ] Factorio
- [x] FF1
- [x] FFMQ
- [x] Heretic
- [x] HK
- [x] Hylics 2
- [x] KH2
- [x] LADX
- [x] Landstalker
- [x] Lingo
- [x] Lufia II
- [x] Meritous
- [x] Minecraft
- [x] MMBN3
- [x] Muse Dash
- [x] Noita
- [x] OOT
- [x] OC2!
- [x] Pokemon Red and Blue
- [x] Raft
- [x] Rogue Legacy
- [x] ROR2
- [x] SA2B
- [x] SC2
- [x] Shivers
- [x] SM
- [x] SM64
- [x] SMW
- [x] SMZ3
- [x] SOE
- [x] STS
- [x] Stardew Valley
- [ ] Subnautica
- [x] Terraria
- [x] TS
- [x] TLOZ
- [x] Undertale
- [x] V6
- [x] Wargroove
- [x] The Witness
- [x] Zillion

### CODE REVIEW

**Review by agilbert1412 — APPROVED** [review]
Approved for DLC Quest and Stardew Valley

**Review by BadMagic100 — APPROVED** [review]
Approved for HK

File: worlds/noita/docs/en_Noita.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+The [player options page for this game](../player-options) contains all the options you need to configure and export a
 config file.
 
 ## What does randomization do to this game?
 
 Noita is a procedurally generated action roguelike. During runs in Noita you will find potions, wands, spells, perks,
 chests, etc. Shop items, chests/hearts hidden in the environment, and pedestal items will be replaced with location
-checks. Orbs and boss drops can give location checks as well, if they are enabled in the settings.
+checks. Orbs and boss drops can give location checks as well, if they are enabled in the options.
```
Comment (ScipioWright) [inline worlds/noita/docs/en_Noita.md:12]:
```suggestion
checks. Orbs and boss drops can give location checks as well, if their respective options are enabled.
```

**Review by ScipioWright — CHANGES_REQUESTED** [review]

File: worlds/noita/docs/en_Noita.md (hunk 1 above)
Comment (ReverM) [reply worlds/noita/docs/en_Noita.md:12]: That is a better phrasing, definitely

**Review by ScipioWright — APPROVED** [review]

**Comment by ScipioWright** [comment]
Approved for Noita

**Comment by ReverM** [comment]
To keep track of which game have been reviewed heres a list :p

- [ ] Adventure
- [ ] LTTP
- [ ] Archipidle
- [ ] Sudoku
- [ ] Blasphemous
- [ ] Bumper Stickers
- [ ] Checksfinder
- [ ] Clique
- [ ] DS3
- [ ] DKC3
- [ ] DLC Quest
- [ ] DOOM
- [ ] DOOM II
- [ ] Factorio
- [ ] FF1
- [ ] FFMQ
- [ ] Heretic
- [ ] HK
- [ ] Hylics 2
- [ ] KH2
- [ ] LADX
- [ ] Landstalker
- [ ] Lingo
- [ ] Lufia II
- [ ] Meritous
- [ ] Minecraft
- [ ] MMBN3
- [ ] Muse Dash
- [ ] Noita
- [ ] OOT
- [ ] OC2!
- [ ] Pokemon Emerald
- [ ] Pokemon Red and Blue
- [ ] Raft
- [ ] Rogue Legacy
- [ ] ROR2
- [ ] SA2B
- [ ] SC2
- [ ] Shivers
- [ ] SM
- [ ] SM64
- [ ] SMW
- [ ] SMZ3
- [ ] SOE
- [ ] STS
- [ ] Stardew Valley
- [ ] Subnautica
- [ ] Terraria
- [ ] TS
- [ ] TLOZ
- [ ] Undertale
- [ ] V6
- [ ] Wargroove
- [ ] The Witness
- [ ] Zillion

**Review by hatkirby — APPROVED** [review]
Approved for Lingo.

**Comment by Daivuk** [comment]
Approved for 
- doom_1993
- doom_ii
- heretic

File: worlds/witness/docs/en_The Witness.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ This includes symbols such as "Dots", "Black/White Squares", "Colored Squares",
 
 Alternatively (or additionally), you can play "Door shuffle", where some doors won't open until you receive their "key".
 
-Receiving lasers as items is also a possible setting.
+Receiving lasers as items is also a possible option.
```
Comment (NewSoupVi) [inline worlds/witness/docs/en_The Witness.md:28]: Oooo I don't like this, this massively overlaps with the colloquial definition of "option"
Maybe this can be reworded to something like "is also an option you can choose"?

(Tbh this is part of the reason I would've preferred "settings" as the new catch-all term, but eh)

File: worlds/subnautica/docs/setup_en.md [hunk 3]
```diff
@@ -19,7 +19,7 @@
 Use the connect form in Subnautica's main menu to enter your connection information to connect to an Archipelago multiworld.
 Connection information consists of:
  - Host: the full url that you're trying to connect to, such as `archipelago.gg:38281`.
- - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when creating your settings.
+ - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when creating your options.
```
Comment (ScipioWright) [inline worlds/subnautica/docs/setup_en.md:22]:
```suggestion
 - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when you chose your options.
```

File: worlds/witness/docs/setup_en.md [hunk 4]
```diff
@@ -43,4 +43,4 @@ The Witness has a fully functional map tracker that supports auto-tracking.
 3. Click on the "AP" symbol at the top.
 4. Enter the AP address, slot name and password. 
 
-The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It will hide checks & adjust logic accordingly.
+The rest should take care of itself! Items and checks will be marked automatically, and it even knows your options - It will hide checks & adjust logic accordingly.
```
Comment (NewSoupVi) [inline worlds/witness/docs/setup_en.md:46]: Ugh. Yeah, another instance where I'm really frustrated by the ambiguity of the word "option". But this one is more acceptable. I would personally probably also reword this to say something like "the options you picked", but it's not as necessary imo

File: worlds/subnautica/docs/setup_en.md (hunk 3 above)
Comment (ReverM) [reply worlds/subnautica/docs/setup_en.md:22]:
```suggestion
 - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when choosing your options.
```

**Review by NewSoupVi — COMMENTED** [review]
Meh, it's okay I guess. I'm realising now I really do not like this change and would've absolutely campaigned for **settings** to be the accepted term if I had thought about it

**Comment by ScipioWright** [comment]
You can probably just check off the following, which have the same boilerplate text regarding the options page as most other games:
Archipidle, Sudoku, Blasphemous, Bumper Stickers, Checksfinder, Clique, DKC3, DLC Quest, Doom, Doom 2, FFMQ, Heretic, Hylics 2, KH2, Lingo, Lufia 2, Minecraft, MMBN3, Raft, Rogue Legacy, SA2B, Shivers, Super Metroid, SMW, SMZ3, Secret of Evermore, Slay the Spire, Timespinner, Zelda 1, Undertale, VVVVVV, Wargroove, and Zillion

File: worlds/witness/docs/setup_en.md (hunk 4 above)
Comment (ReverM) [reply worlds/witness/docs/setup_en.md:46]:
```suggestion
The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.
```

**Review by FlySniper — APPROVED** [review]
Approved for Wargroove.

File: worlds/witness/docs/en_The Witness.md (hunk 2 above)
Comment (ReverM) [reply worlds/witness/docs/en_The Witness.md:28]:
```suggestion
You can also chose to receive lasers as items as an option.
``` how does this sound?

File: worlds/subnautica/docs/setup_en.md (hunk 3 above)
Comment (ScipioWright) [reply worlds/subnautica/docs/setup_en.md:22]: Either way should work. It's up to Berserker here, anyway

Comment (ReverM) [reply worlds/subnautica/docs/setup_en.md:22]: yep

File: worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ The goal is to beat the final mission: 'All In'. The config file determines whic
 ## Which of my items can be in another player's world?
 
 By default, any of StarCraft 2's items (specified above) can be in another player's world. See the
-[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_settings/en)
+[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_options/en)
```
Comment (Exempt-Medic) [inline worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md:33]: advanced_options doesn't exist

Comment (ReverM) [reply worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md:33]: Hmm good catch, I think that page needs to be changed actually

File: worlds/witness/docs/setup_en.md (hunk 4 above)
Comment (NewSoupVi) [reply worlds/witness/docs/setup_en.md:46]: Yeah I like that :3

File: worlds/witness/docs/en_The Witness.md (hunk 2 above)
Comment (NewSoupVi) [reply worlds/witness/docs/en_The Witness.md:28]: Sure, either of those work for me, I probably thing Rever's is better? Otherwise someone might think they can do some in-game action for it or sth

File: worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md (hunk 5 above)
Comment (ReverM) [reply worlds/sc2wol/docs/en_Starcraft 2 Wings of Liberty.md:33]:
```suggestion
[Advanced YAML Guide](https://archipelago.gg/tutorial/Archipelago/advanced_settings/en)
```

File: worlds/witness/docs/en_The Witness.md (hunk 2 above)
Comment (ReverM) [reply worlds/witness/docs/en_The Witness.md:28]: Scipio caught a typo tho o:

**Review by NewSoupVi — APPROVED** [review]
Well, I don't agree with the mission objective, but the mission was carried out successfully for The Witness, and if "options" is gonna be the new word (bleh), so be it. I'll use whatever daddy AP wants me to use

**Review by TRPG0 — APPROVED** [review]
Looks good for Blasphemous and Hylics 2.

**Comment by Dinopony** [comment]
Approved for Landstalker

**Review by Dinopony — APPROVED** [review]
Approved for Landstalker

File: worlds/witness/docs/en_The Witness.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ This includes symbols such as "Dots", "Black/White Squares", "Colored Squares",
 
 Alternatively (or additionally), you can play "Door shuffle", where some doors won't open until you receive their "key".
 
-Receiving lasers as items is also a possible setting.
+You can also choose to receive lasers as items as an option.
```
Comment (ReverM) [inline worlds/witness/docs/en_The Witness.md:28]:
```suggestion
You can also set lasers to be items you can receive.
```

File: worlds/adventure/docs/en_Adventure.md [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
 
-## Where is the settings page?
-The [player settings page for Adventure](../player-settings) contains all the options you need to configure and export a config file.
+## Where is the options page?
+The [player options page for Adventure](../player-options) contains all the options you need to configure and export a config file.
 
 ## What does randomization do to this game?
 Adventure items may be distributed into additional locations not possible in the vanilla Adventure randomizer.  All
-Adventure items are added to the multiworld item pool.  Depending on the settings, dragon locations may be randomized,
+Adventure items are added to the multiworld item pool.  Depending on the options, dragon locations may be randomized,
```
Comment (ReverM) [inline worlds/adventure/docs/en_Adventure.md:8]:
```suggestion
Adventure items are added to the multiworld item pool.  Depending on the `dragon_rando_type` value, dragon locations may be randomized,
```

File: worlds/factorio/docs/setup_en.md [hunk 8]
```diff
@@ -148,11 +148,11 @@ For more information about the commands you can use, see the [Commands Guide](/t
 By default, peaceful mode is disabled. There are two methods to enable peaceful mode:
 
 ### By config file
-You can specify Factorio game settings such as peaceful mode and terrain and resource generation parameters in your
-config .yaml file by including the `world_gen` setting. This setting is currently not supported by the web UI, so you'll
+You can specify Factorio game options such as peaceful mode and terrain and resource generation parameters in your
```
Comment (ReverM) [inline worlds/factorio/docs/setup_en.md:151]: Should this one be reverted actually?

File: worlds/pokemon_rb/docs/setup_en.md [hunk 9]
```diff
@@ -114,5 +114,5 @@ Pokémon Red and Blue has a fully functional map tracker that supports auto-trac
 3. Click on the "AP" symbol at the top.
 4. Enter the AP address, slot name and password. 
 
-The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It
+The rest should take care of itself! Items and checks will be marked automatically, and it even knows your options - It
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_en.md:117]: Might revert?

File: worlds/witness/docs/setup_en.md [hunk 10]
```diff
@@ -43,4 +43,4 @@ The Witness has a fully functional map tracker that supports auto-tracking.
 3. Click on the "AP" symbol at the top.
 4. Enter the AP address, slot name and password. 
 
-The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It will hide checks & adjust logic accordingly.
+The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.
```
Comment (ReverM) [inline worlds/witness/docs/setup_en.md:46]: Might revert?

**Review by wildham0 — APPROVED** [review]
Looks good for FF1

**Comment by LegendaryLinux** [comment]
ArchipIDLE changes are fine.

File: worlds/soe/docs/multiworld_en.md [hunk 11]
```diff
@@ -23,8 +23,8 @@ guide: [Basic Multiworld Setup Guide](/tutorial/Archipelago/setup/en)
 
 ### Where do I get a config file?
 
-The Player Settings page on the website allows you to configure your personal settings and export a config file from
-them. Player settings page: [Secret of Evermore Player Settings PAge](/games/Secret%20of%20Evermore/player-settings)
+The Player Options page on the website allows you to configure your personal options and export a config file from
+them. Player options page: [Secret of Evermore Player Options PAge](/games/Secret%20of%20Evermore/player-options)
```
Comment (black-sliver) [inline worlds/soe/docs/multiworld_en.md:27]:
```suggestion
them. Player options page: [Secret of Evermore Player Options Page](/games/Secret%20of%20Evermore/player-options)
```

**Review by black-sliver — CHANGES_REQUESTED** [review]
Only checked SoE. Found this typo, otherwise SoE change looks fine.

**Review by LegendaryLinux — APPROVED** [review]

**Review by black-sliver — APPROVED** [review]
Approving the changes to SoE

**Review by ThePhar — APPROVED** [review]
RL/Clique/HK looks fine.

File: worlds/witness/docs/en_The Witness.md (hunk 6 above)
Comment (BootsinSoots) [reply worlds/witness/docs/en_The Witness.md:28]: I like that second one better

**Review by nicholassaylor — APPROVED** [review]
While not a codeowner for either game, I am giving my approval for DS3 and Risk of Rain 2 due to my familiarity with both randomizers. Both games' set-up guides are simply copy-paste changes and both game pages have no issues.

File: worlds/adventure/docs/en_Adventure.md (hunk 7 above)
Comment (JusticePS) [reply worlds/adventure/docs/en_Adventure.md:8]: Looks good.  No reason not to specify it unless additional options interfere it, and I don't think they do.

**Review by JusticePS — APPROVED** [review]
Adventure is good (including the additional change is also good)

**Review by toasterparty — APPROVED** [review]
Looks like my game was the only one with a link to "weighted settings". I approve *with the assumption* `../../../../weighted-options` is the correct path for "weighted options".

**Comment by Ziktofel** [comment]
For SC2 See also #2954

**Review by Ziktofel — APPROVED** [review]

**Review by Seldom-SE — APPROVED** [review]
Looks good for Terraria

**Review by FelicitusNeko — APPROVED** [review]

**Review by N00byKing — APPROVED** [review]

**Review by espeon65536 — APPROVED** [review]

**Review by NewSoupVi — APPROVED** [review]

File: worlds/witness/docs/setup_en.md (hunk 10 above)
Comment (ReverM) [inline worlds/witness/docs/setup_en.md:46]:
```suggestion
The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It will hide checks & adjust logic accordingly.
```

**Review by NewSoupVi — APPROVED** [review]

**Review by ScipioWright — APPROVED** [review]

**Comment by Br00ty** [comment]
@black-sliver DS3 changes look good!

**Comment by ThePhar** [comment]
Since the SDV 5.x merge, this now needs to be resolved

image

**Review by Br00ty — APPROVED** [review]
DS3 looks good

**Review by DeamonHunter — APPROVED** [review]
Muse Dash is good.

**Review by agilbert1412 — APPROVED** [review]
Re-reviewed Stardew Valley

**Review by black-sliver — APPROVED** [review]
